docs/HowToUseMultiHeap.md

```diff
@@ -0,0 +1,13 @@
+# How to use(add) Multi-heap Region
+
```

Comment:
You had better write when we use this.
1. continous physical ram but want to split region
2. separated physical ram
blah blah

Reply:
done